.ci/.jenkins_windows.yml

```diff
@@ -0,0 +1,29 @@
+# This is the limited list of matrix builds in Windows, to be triggered on a PR basis
```

Comment:
This diverges a bit from what we have right now. Presently, we have [Python versions](https://github.com/elastic/apm-agent-python/blob/master/.ci/.jenkins_python.yml) and [frameworks](https://github.com/elastic/apm-agent-python/blob/master/.ci/.jenkins_framework.yml) defined in separate files. Then we're using [the Parallel Tasks Generator](https://github.com/elastic/apm-pipeline-library/blob/master/src/co/elastic/matrix/DefaultParallelTaskGenerator.groovy) to create the matrix and define the steps. This seems as though [it partially re-implements that logic](https://github.com/elastic/apm-agent-python/pull/746/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R421).

I think that this divergence should be fine so long as we aren't expecting large numbers of combinations, which might be difficult to maintain in a single file such as this.

Do we anticipate this will be a problem in the future? (I don't have really strong preferences about which approach is taken. I am more curious about the reasoning here than anything.)

Reply:
For Windows, we don't plan to have a matrix anywhere as extensive as the one one Linux. For now, we only have one element on the `FRAMEWORK` axis (`none`, which selects all tests that are framework-independent). We might want to add a couple of Windows-specific ones later on, e.g. MS-SQL, but that's about it.

Reply:
Cool! That resolves my concerns then. :) Thanks, @beniwohli